app/views/course/achievements/index.html.slim

```diff
@@ -4,10 +4,11 @@ table.table.achievement-list.table-hover
   thead
     tr
       th
+      th = t('.title')
       th = t('.description')
       th = t('.requirements')
       - if can?(:manage, Course::Achievement)
-      th = t('.published')
+        th = t('.published')
       th
   tbody
-    = render @achievements
\ No newline at end of file
+    = render @achievements.includes(:conditions)
```

Comment:
Is there a way to hook into what `load_resource` does? For example to use `.includes` or `.order` on a collection before it's passed to the view?